dev/tools/dartdoc.dart

```diff
@@ -36,15 +35,18 @@ Future<Null> main(List<String> args) async {
   if (path.basename(Directory.current.path) == 'tools')
     Directory.current = Directory.current.parent.parent;
 
-  final RawVersion version = new RawVersion('VERSION');
+  final ProcessResult flutter = Process.runSync('flutter', <String>[]);
+  final File versionFile = new File('version');
+  if (flutter.exitCode != 0 || !versionFile.existsSync())
+    throw new Exception('Failed to determine Flutter version.');
+  final String version = versionFile.readAsStringSync();
 
   // Create the pubspec.yaml file.
-  final StringBuffer buf = new StringBuffer('''
-name: Flutter
-homepage: https://flutter.io
-version: $version
-dependencies:
-''');
+  final StringBuffer buf = new StringBuffer();
+  buf.writeln('name: Flutter');
+  buf.writeln('homepage: https://flutter.io');
+  buf.writeln('version: $version');
```

Comment:
The Travis doc script is complaining about "version: unknown"

Reply:
fixed